### ISSUE TITLE
NMDC runtime failure when there are indexes for duplicate IDs

### ISSUE THREAD

**Issue by shreddd** [microbiomedata/nmdc-runtime#242]
runtime seems to creash if it finds duplicate IDs (possibly when it creates indexes)?

We should have a SOP for recovery or figure out if there is a way to address this during startup (dy dropping / recreating indexes etc.)

-Shreyas

**Comment by turbomam** [comment]
LinkML and check-jsonschema both provide validation over duplicate IDs within containers,if you want to address this at a file (as opposed to database) level.

**Comment by shreddd** [comment]
Also see #240

**Comment by dwinston** [comment]
@shreddd I've added a "needs reproduction" label on https://github.com/microbiomedata/nmdc-runtime/issues/240 . I added some messaging that's less cryptic, but I think it's best to fail when there are duplicate IDs because there is runtime logic (i.e. mongo `find_one` and `update_one` ops) that will have unexpected behavior when 'id' is not guaranteed to be unique across the relevant collections.

Thus, I recommend closing this issue and focusing on #240 to address the concern.
